### DISCUSSION TITLE
API image creation path

### DISCUSSION THREAD

**Discussion by lionelgoodboy** [lllyasviel/Fooocus#3419] [Q&A] [answered]
hi everyone! I instaled gradio client and tested same API , but no success creating the images here. 
There is no error, i call it using windows prompt, for example: python test.py

But, how to create the images using those apis? No images has been created in output folder. And I tryed a lot of function index of the API.

from gradio_client import Client

client = Client("http://127.0.0.1:7865/")
result = client.predict(
				fn_index=0
)
print(result)

Thanks for any help! ;)

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/issues/3422#issuecomment-2264065880
